velox/docs/functions/spark/aggregate.rst

```diff
@@ -24,14 +24,13 @@ General Aggregate Functions
 .. spark:function:: bloom_filter_agg(hash, estimatedNumItems, numBits) -> varbinary
 
     Creates bloom filter from input hashes and returns it serialized into VARBINARY.
-    The caller is expected to apply xxhash64 function to input data before calling bloom_filter_agg.
 
     For example, 
         bloom_filter_agg(xxhash64(x), 100, 1024)
     In Spark implementation, ``estimatedNumItems`` and ``numBits`` are used to decide the number of hash functions and bloom filter capacity.
     In Velox implementation, ``estimatedNumItems`` is not used.
 
-    ``hash`` cannot be null.
+    ``hash`` can be null.
```

Comment:
L44 needs updating as well. Please, do review all the documentation to make sure it is updated to accommodate changes.